output.R

```diff
@@ -139,11 +139,15 @@ choose_mode <- function(title = "Please choose the output mode") {
   return(comp)
 }
 
-choose_slurmConfig_priority_standby <- function(high_mem = FALSE,
-                                                title = "Please enter the slurm mode, uses priority if empty") {
-  slurm_options <- c(if (! high_mem) c("--qos=priority", "--qos=short", "--qos=standby"),
+choose_slurmConfig_priority_standby <- function(title = "Please enter the slurm mode, uses priority if empty",
+                                                slurmExceptions = NULL) {
+  slurm_options <- c("--qos=priority", "--qos=short", "--qos=standby",
                      "--qos=priority --mem=8000", "--qos=short --mem=8000",
                      "--qos=standby --mem=8000", "--qos=priority --mem=32000", "direct")
+  if (!is.null(slurmExceptions)) {
+    slurm_options <- grep(slurmExceptions, slurm_options, value = TRUE)
+  }
+  if (length(slurm_options) == 1) return(slurm_options[[1]])
```

Comment:
I like how you catch the length 1 case, but I wonder if for safety reasons, one should make sure that `direct` is always offered. So replace Line 148 with
```
slurm_options <- unique(c("direct", grep(slurmExceptions, slurm_options, value = TRUE))
```
This way, also the second comment would be fine…

Reply:
`direct` is now always available following your suggestion.